Druid/DRScope.class.st

```diff
@@ -2,9 +2,164 @@ Class {
 	#name : 'DRScope',
 	#superclass : 'Object',
```

Comment:
I think that this class is saving many things that could be generated in a method: _dependent vs independent variables_.